addons/im_livechat/static/tests/mock_server/mock_models/discuss_channel.js

```diff
@@ -27,6 +28,11 @@ export class DiscussChannel extends mailModels.DiscussChannel {
         }
         return super.action_unfollow(...arguments);
     }
+
+    _channel_internal_user_info() {
+        return super._channel_internal_user_info(...arguments).concat(["livechat_node"]);
```

Comment:
typo: note

Currently this code is useless until we also write a test editing the note.